### ISSUE TITLE
incorrect order of data points in plot_forest

### ISSUE THREAD

**Issue by zhiyuan-s** [psychmeta/psychmeta#122]
When using plot_forest, the order in the y axis is not correct. It treats the y coordinates as strings, not integers. For example, if the references are 1 - 20, the current code would sort them as 1, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 2, 20, 3, 4, 5, 6, 7, 8, 9.

**Comment by bwiernik** [comment]
Can you give a reproducible example?

**Comment by zhiyuan-s** [comment]
https://sites.psu.edu/yuweili/research-methods/psychmeta/
I'm learning psychmeta using this tutorial.

**Comment by jadahlke** [comment]
I agree that the y-axis ordering is something we can improve upon (e.g., by allowing users to designate the sample order by supplying sample_id as a factor), but I'm not sure that this is unexpected behavior based on our original concept for the function. We had intended for users to supply sample_id values that identify the sources of the statistics being meta-analyzed (e.g. "Smith et al. (1998)"), and the y-axis would display these in alphabetical order. The use of numeric IDs for studies is not really what we had in mind when applying this function in real meta-analyses. We'll work on making this more flexible.
